Makefile

```diff
@@ -315,9 +311,9 @@ fuzz:
 # =========
 
 #? fmt: Format source code and fix imports
-fmt: $(GOIMPORTS_BIN)
+fmt:
 	@$(call print, "Fixing imports.")
-	gosimports -w $(GOFILES_NOVENDOR)
+	$(GOCC) tool --modfile $(TOOLS_MOD)  $(GOIMPORTS_PKG) -w $(GOFILES_NOVENDOR)
```

Comment:
This line will break the `go.work`,
```sh
> make lint
 Linting source.
go tool --modfile ./tools/go.mod github.com/golangci/golangci-lint/v2/cmd/golangci-lint run -v 
go: -modfile cannot be used in workspace mode
make: *** [lint] Error 1
```

Reply:
removed the modfile reference but still kept the usage of go tool.